ddtrace/profiling/exporter/pprof.py

```diff
@@ -230,8 +236,21 @@ def _get_program_name(default="-"):
         return program_name
 
     @staticmethod
-    def _stack_event_group_key(event):
-        return (event.thread_id, event.thread_native_id, str(event.thread_name), tuple(event.frames), event.nframes)
+    def _get_trace_id(event):
+        if event.trace_ids:
+            return str(list(sorted(event.trace_ids))[0])
+        return ""
+
+    def _stack_event_group_key(self, event):
+        # If multiple traces were active, we pick only one :(
```

Comment:
Partly, but more a `pprof` limitation. You can't have an array of values for a label, so the label `trace_id` can only have one value. Or we would have to do `trace_id1`, trace_id2`, which is not really elegant.

The final goal is to refine the data collection to have the right trace — there should be only one active, right? — but for now we don't support coroutines correctly e.g. gevent or asyncio. So it might happens.